The version of quilt we have currently doesn't have a "remove" command but
it has a "revert" command, so the bash completion should handle that.
---
 bash_completion |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- quilt-0.48.orig/bash_completion
+++ quilt-0.48/bash_completion
@@ -97,7 +97,7 @@ _quilt_completion()
     # quilt sub commands 
     cmds='add annotate applied delete diff edit files fold fork graph \
           grep header import mail new next patches pop previous push refresh \
-	  remove rename series setup snapshot top unapplied'
+	  revert rename series setup snapshot top unapplied'
 
     # if no command were given, complete on commands
     if [[ $COMP_CWORD -eq 1 ]] ; then
@@ -253,7 +253,7 @@ _quilt_completion()
 		;;
 	   esac
 	   ;;
-	remove)
+	revert)
 	   case $prev in
 	     -P)
  	        COMPREPLY=( $( compgen -W "$(quilt applied)" -- $cur ) )
